osquery/tables/applications/posix/docker.cpp

```diff
@@ -1096,6 +1096,84 @@ QueryData genImageLayers(QueryContext& context) {
   return results;
 }
 
+/**
+ * @brief Image history extractor for docker_image_history table
+ */
+void getImageHistory(std::string image_id, QueryData& results) {
```

Comment:
```suggestion
void getImageHistory(const std::string& image_id, QueryData& results) {
```

Reply:
Thanks @theopolis . Fixed this and another function with similar signature.